docs/products/clickhouse/howto/use-play.rst

```diff
@@ -0,0 +1,14 @@
+Use ClickHouse® play UI
+========================
+
+ClickHouse® includes a built-in user interface for running SQL queries. You can access it through HTTPS protocol. Follow these steps to use it:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_ and select your ClickHouse service.
+#. In the *Overview* tab find *Connection information* and select **ClickHouse HTTPS**
+#. Copy Service URI and use it to open a page in the browser with the URL ``your-full-service-uri/play``.
+#. Set the *name* and the *password* of the user of whose behalf you want to run the queries.
```

Comment:
Set the *name* and the *password* of the user ~of~ on whose behalf you want to run the queries.